### PR TITLE
11.0 mig server environment

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

**Comment by TDu** [comment]
Travis is green :smiley: 
But code coverage is complaining about 'Missing base report'
Anyone knows how to fix this ?

**Review by grindtildeath — APPROVED** [review]
Works well :+1:

**Comment by TDu** [comment]
Code coverage is fixed, thanks invisible hand.
Last commit fixes a bug when displaying the view from Settings > Technical > Server Environment

**Review by guewen — APPROVED** [review]

File: server_environment/__init__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+##############################################################################
```
Comment (simahawk) [inline server_environment/__init__.py:1]: shouldn't we remove these headers from __init__.py and shorten the other ones?

**Review by simahawk — APPROVED** [review]
LGTM, just a not on copy headers.

**Comment by yvaucher** [comment]
As stated here https://docs.google.com/spreadsheets/d/1Xg95cW4TFMf_Lo5i_CZC_qOOfN8RgxPRc0LJTLTkdUI/edit?usp=sharing

could we create a project for server-environement modules ?

OCA/server-env for

mail_environment
server_environment
server_environment_files_sample
server_environment_ir_config_parameter

**Comment by simahawk** [comment]
replaced by https://github.com/OCA/server-env/pull/4

Please, close this one

**Pull request closed without merging**